PWG/EMCAL/EMCALbase/AliAnalysisTaskEmcalEmbeddingHelperData.cxx

```diff
@@ -0,0 +1,97 @@
+#include <vector>
+
+#include "AliAnalysisTaskEmcalEmbeddingHelper.h"
+#include "AliAnalysisTaskEmcalEmbeddingHelperData.h"
+#include "AliAODEvent.h"
+#include "AliESDEvent.h"
+#include "AliVAODHeader.h"
+#include "AliEmcalList.h"
+
+/// \cond CLASSIMP
+ClassImp(AliAnalysisTaskEmcalEmbeddingHelperData);
+/// \endcond
+
+AliAnalysisTaskEmcalEmbeddingHelperData::AliAnalysisTaskEmcalEmbeddingHelperData():
+  AliAnalysisTaskEmcalEmbeddingHelper(),
+  fHistManager(0),
+  fPtSelectionMin(),
+  fPtSelectionMax()
+{
+}
+
+
+AliAnalysisTaskEmcalEmbeddingHelperData::AliAnalysisTaskEmcalEmbeddingHelperData(const char *name):
+  AliAnalysisTaskEmcalEmbeddingHelper(name),
+  fHistManager(name),
+  fPtSelectionMin(),
+  fPtSelectionMax()
+{
+}
+
+AliAnalysisTaskEmcalEmbeddingHelperData::~AliAnalysisTaskEmcalEmbeddingHelperData()
+{
+}
+
+void AliAnalysisTaskEmcalEmbeddingHelperData::UserCreateOutputObjects() 
+{
+
+  // run function from embedding helper base class
+  AliAnalysisTaskEmcalEmbeddingHelper::UserCreateOutputObjects();
```

Comment:
I'm not 100% sure if this would work, but created the histogram below duplicates a bunch of code. It might work to define the hist, add it to the hist manager, and then call the base class? Then you could remove adding the histograms, as well as the `PostData`